crates/ruma-events/src/beacon.rs

```diff
@@ -0,0 +1,43 @@
+//! Types for the `org.matrix.msc3489.beacon` event, the unstable version of
+//! `m.beacon`.
+//!
+//! [MSC3489]: https://github.com/matrix-org/matrix-spec-proposals/pull/3489
```

Comment:
This is a link reference. It should be included somewhere in the text to appear. For example:

```rust
//! Types for the `org.matrix.msc3489.beacon` event, the unstable version of
//! `m.beacon` ([MSC3489]).
```